pkg/services/sqlstore/migrator/migrator.go

```diff
@@ -208,6 +209,13 @@ func (mg *Migrator) run() (err error) {
 
 		err := mg.InTransaction(func(sess *xorm.Session) error {
 			err := mg.exec(m, sess)
+			// if we get an sqlite busy/locked error, sleep 100ms and try again
```

Comment:
I've seen this with WAL enabled too if memory serves. Are you suggesting adding mention of that to the comment?

Reply:
I had considered that option, but actually I think I was more mentioning it in case you weren't aware of that config. So no, I don't think including it in the comment is necessary.